### ISSUE TITLE
Search within command templates results in Uncaught TypeError: Argument 2 passed to gipfl

### ISSUE THREAD

**Issue by RobbieTheK** [Icinga/icingaweb2-module-director#2422]
## Expected Behavior
Search within command templates results in` Uncaught TypeError: Argument 2 passed to gipfl`

## Current Behavior
```
Uncaught TypeError: Argument 2 passed to gipfl\IcingaWeb2\Zf1\Db\FilterRenderer::applyToQuery() must be an instance of Zend_Db_Select, instance of Icinga\Data\SimpleQuery given, called in /usr/share/icingaweb2/modules/incubator/vendor/gipfl/icingaweb2/src/Table/QueryBasedTable.php on line 98 and defined in /usr/share/icingaweb2/modules/incubator/vendor/gipfl/icingaweb2/src/Zf1/Db/FilterRenderer.php:47
Stack trace:
#0 /usr/share/icingaweb2/modules/incubator/vendor/gipfl/icingaweb2/src/Table/QueryBasedTable.php(98): gipfl\IcingaWeb2\Zf1\Db\FilterRenderer::applyToQuery()
#1 /usr/share/icingaweb2/modules/incubator/vendor/gipfl/icingaweb2/src/Table/QueryBasedTable.php(241): gipfl\IcingaWeb2\Table\QueryBasedTable->search()
#2 /usr/share/icingaweb2/modules/incubator/vendor/gipfl/icingaweb2/src/Table/QueryBasedTable.php(256): gipfl\IcingaWeb2\Table\QueryBasedTable->initializeOptionalQuickSearch()
#3 /usr/share/icingaweb2/modules/director/library/Director/Web/Controller/ObjectController.php(228): gipfl\IcingaWeb2\Tabl

#0 [internal function]: Icinga\Application\Web->Icinga\Application\{closure}()
#1 {main}
```

## Steps to Reproduce (for bugs)
Director --> Commands --> Command Templates --> Select a Command Template that is not in use --> Fields, then search in top right

## Your Environment
Director version (System - About): 1.8.1
Icinga Web 2 version and modules (System - About): 2.9.3
Icinga 2 version (icinga2 --version):
icinga2 - The Icinga 2 network monitoring daemon (version: 2.13.1-1)

Copyright (c) 2012-2021 Icinga GmbH (https://icinga.com/)
License GPLv2+: GNU GPL version 2 or later https://gnu.org/licenses/gpl2.html
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

System information:
Platform: Fedora
Platform version: 34 (Server Edition)
Kernel: Linux
Kernel version: 5.13.19-200.fc34.x86_64
Architecture: x86_64

Build information:
Compiler: GNU 11.2.1
Build host: unknown
OpenSSL version: OpenSSL 1.1.1l FIPS 24 Aug 2021

Application information:

General paths:
Config directory: /etc/icinga2
Data directory: /var/lib/icinga2
Log directory: /var/log/icinga2
Cache directory: /var/cache/icinga2
Spool directory: /var/spool/icinga2
Run directory: /run/icinga2

Old paths (deprecated):
Installation root: /usr
Sysconf directory: /etc
Run directory (base): /run
Local state directory: /var

Internal paths:
Package data directory: /usr/share/icinga2
State path: /var/lib/icinga2/icinga2.state
Modified attributes path: /var/lib/icinga2/modified-attributes.conf
Objects path: /var/cache/icinga2/icinga2.debug
Vars path: /var/cache/icinga2/icinga2.vars
PID path: /run/icinga2/icinga2.pid
Operating System and version: Fedora 34
Webserver, PHP versions: Apache, PHP 7.4.24

![icingaerror](https://user-images.githubusercontent.com/21204619/140087687-bd84c195-6d2f-44de-9ff9-29dd69f82a21.PNG)

**Comment by Thomas-Gelf** [comment]
This [fix](https://github.com/gipfl/icingaweb2/commit/1a4612db4f49ec21d37c02cf5dc59aa2b07b3987) should address this, I'll release a related new incubator module.

**Comment by Thomas-Gelf** [comment]
Fixed with 2f32657
